bundles/org.aposin.mergeprocessor/src/org/aposin/mergeprocessor/view/dashboard/MergeUnitViewerComparator.java

```diff
@@ -55,7 +55,7 @@ int getColumn() {
 	 */
 	@Override
 	public int compare(Viewer viewer, Object e1, Object e2) {
-		if (e1 == e2) {
```

Comment:
Same problem with spaces instead of tabs.

Additionally, a simple JUnit test for MergeUnitViewerComparator would be nice to cover this change.